dev/PushNotifications/PushNotificationsLongRunningTask/packages.config

```diff
@@ -1,8 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.FrameworkUdk.amd64fre" version="10.0.22453.1002-210908-1649.ni-release-svc-reunion2110" targetFramework="native" />
-  <package id="Microsoft.FrameworkUdk.arm64fre" version="10.0.22453.1002-210908-1649.ni-release-svc-reunion2110" targetFramework="native" />
-  <package id="Microsoft.FrameworkUdk.x86fre" version="10.0.22453.1002-210908-1649.ni-release-svc-reunion2110" targetFramework="native" />
+  <package id="Microsoft.FrameworkUdk.amd64fre" version="10.0.22453.1006-210916-1712.ni-release-svc-reunion2110" targetFramework="native" />
```

Comment:
why update only the LRP project, please update all the other projects where ever we are using frameworkudk?

Reply:
Do you know what are these other projects?

Reply:
WindowsAppRuntime_DLL?

Reply:
As stated in the description, this was done when the UDK was ingested.

Reply:
Thank you.